includes/redis-connect-helper.php

```
@@ -32,159 +32,228 @@
+ * @return array Associative array containing 'host' and 'port'.
+ */
+function wppo_parse_redis_node( $node ) {
+	if ( strpos( $node, '[' ) === 0 ) {
+		$port_start = strpos( $node, ']:' );
+		if ( false !== $port_start ) {
+			$host = substr( $node, 1, $port_start - 1 );
+			$port = (int) substr( $node, $port_start + 2 );
+		} else {
+			$host = trim( $node, '[]' );
+			$port = 26379;
+		}
```

PHPMD (2.15.0)

[error] The method wppo_parse_redis_node uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)